### ISSUE TITLE
Random problem with buffering of videos

### ISSUE THREAD

**Issue by rako233** [yuliskov/SmartTubeNext#692]
SmartTubeNext has some issues with buffering of videos on a NVidia ShieldTV. Description: 

1. Not every video is affected. 
2. The problem is not related the codec 
3. The problem is not related to the internet connection. I have no problem to play a certain 4k UHD video with 18mbit/s while 2k videos with 5mbit/s have this problem. A cross check with the Google YT app confirms this. Everything is fine there
4. Instead of building a buffer of 50s the buffer is emptying 
5. I had a similar issue with my web reader and had to delete data like cookies. 

To me it's looking like a change of protocol on the side of Google

**Comment by hamswolo** [comment]
Yup having the same issue on my firestick, random videos buffering, when I back put of app and try again they load. Not app breaking but can be irritating.

**Comment by shravanpn7** [comment]
I'm seeing the same issue on Nvidia shield (latest model) and running STN11.40. I have a 550mbps download speed but as OP mentioned, the buffers empties and video switches to buffering mode multiple times.

Not all videos are affected by this.

I can provide any debug info if needed.

**Comment by Letdown27** [comment]
I also have this issue on my nvidia ShieldTV. Usually, full closing the app and restarting will allow the video to play normally. Unfortunately, however, the issue will soon return on subsequent videos. Seems random but very prevalent.

**Comment by Bale0** [comment]
Another with Nvidia shield on 1GB internet connecting and getting the same buffering on some videos and also after playing a few music videos in a row the app randomly shuts down.

**Comment by Kokkolabs** [comment]
Same here with Shield tv pro 2019.

**Comment by RakeshReddy69** [comment]
Similar issue, when I change accounts random buffer

What worked for me 
Close the app >Settings>apps>SmartTubeNext>Clear Cache

Hope this works for you all too and helps the developer fox this issue in the next update

**Comment by costinmx** [comment]
The issue is widely debated in #658 . Opening more and more identical topics I don't think it s helping the developer, or make the issue resolved faster.

**Comment by M-McFly** [comment]
11.42 has appeared to solve my buffering issue.

**Comment by kwoodall** [comment]
Same problem on my Fire TV 4k... Regular YouTube app doesn't have buffering issues.

**Comment by Bale0** [comment]
Seems fixed now since last update, working perfect for me.
